src/main/java/com/epam/lowcost/controller/UserController.java

```diff
@@ -72,19 +72,18 @@ public String updateUser(@RequestParam Map<String, String> params, Model model)
         if (user == null) {
             model.addAttribute("message", "No such user or it has been deleted!");
         }
-        if(params.get("userUpdate").equals("fromUser")){
+        if (params.get("userUpdate").equals("fromUser")) {
             model.addAttribute("sessionUser", user);
             return "redirect:/tickets/self";
-        }
-        else {
+        } else {
             model.addAttribute("user", user);
             model.addAttribute("message", "User successfully updated");
         }
         return "users";
     }
 
     @PostMapping(value = "registration")
```

Comment:
Somewhere u use endpoint with `/`, somewhere without. why ?

Reply:
returns like `return "users"; ` is direct links to .jsp files.

Reply:
Styled endpoints uniformly.